rails-time-track/spec/controllers/api/histories_controller_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe Api::HistoriesController, type: :controller do
+
+  before do
+    @history =  Project.create(
+      name: "Project Space",
+      client: "Frank",
+      category: "Education",
+      closed: true
+    )
```

Comment:
Similar to the previous comment, we need to add more than one project to test the response